src/js/service/ImportService.js

```diff
@@ -80,6 +80,73 @@
   };
 
   /**
+   * Given an image object and some options, merge the frames with the currently open
+   * Piskel and open it for editing.
+   * @param {!Image} image
+   * @param {!Object} options
+   * @param {!string} options.importType - 'single' if not spritesheet
+   * @param {!number} options.frameSizeX
+   * @param {!number} options.frameSizeY
+   * @param {number} [options.frameOffsetX] only used in spritesheet imports.
+   * @param {number} [options.frameOffsetY] only used in spritesheet imports.
+   * @param {!boolean} options.smoothing
+   * @param {number} [options.frameRate] in frames per second
+   * @param {function} [onComplete]
+   */
+  ns.ImportService.prototype.importFramesFromImage = function (image, options, onComplete) {
+    onComplete = onComplete || Constants.EMPTY_FUNCTION;
+    var importType = options.importType;
+    var frameSizeX = options.frameSizeX;
+    var frameSizeY = options.frameSizeY;
+    var frameOffsetX = options.frameOffsetX;
+    var frameOffsetY = options.frameOffsetY;
+    var smoothing = options.smoothing;
+    var frameRate = typeof options.frameRate !== 'undefined' ?
+        options.frameRate : Constants.DEFAULT.FPS;
+
+    var setPiskelFromFrameImages = function (frameImages) {
+      var piskel = this.createPiskelFromImages_(frameImages, frameSizeX,
+          frameSizeY, smoothing);
+      var mergedPiskel = this.mergePiskels(this.piskelController_.getPiskel(), piskel);
+      this.piskelController_.setPiskel(mergedPiskel);
+      this.previewController_.setFPS(frameRate);
```

Comment:
Looks like the new framerate clobbers the old framerate - and from Piskel's point of view I suppose this is fine since it's only a preview setting.  Still, it seems like an odd choice for this API that adds frames to an existing animation.